_posts/2020-11-13-quarkiverse.adoc

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: Quarkiverse - The Hub of Community Extensions
```

Comment:
is this blog introducing Quarkiverse (the notion of all extensions and tooling around it) or or Quarkiverse Hub the github org to simplify publisihing into quarkiverse ? 

we should pick one to focus on :)